Dockerfile

```diff
@@ -1,6 +1,6 @@
 # syntax=docker/dockerfile:experimental
 
-FROM golang:1.16.12-alpine3.15 as dev
+FROM golang:1.16.5-alpine3.13 as dev
```

Comment:
Probably should not change back to an older version of golang

Reply:
Yes, fixed thank you.
FTR: I faced opencontainers/runc#2151 locally; had to update docker to be able to work with alpine > 3.14 as stated here https://wiki.alpinelinux.org/wiki/Release_Notes_for_Alpine_3.14.0#faccessat2